pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -718,10 +730,17 @@ public CompletableFuture<Void> validateBundleOwnershipAsync(NamespaceBundle bund
                                 if (!owned) {
                                     boolean newAuthoritative = this.isLeaderBroker();
                                     // Replace the host and port of the current request and redirect
-                                    URI redirect = UriBuilder.fromUri(uri.getRequestUri()).host(webUrl.get().getHost())
-                                            .port(webUrl.get().getPort()).replaceQueryParam("authoritative",
-                                                    newAuthoritative).replaceQueryParam("destinationBroker",
-                                                    null).build();
+                                    UriBuilder builder = UriBuilder.fromUri(uri.getRequestUri())
+                                            .host(webUrl.get().getHost())
+                                            .port(webUrl.get().getPort())
+                                            .replaceQueryParam("authoritative", newAuthoritative);
+                                    if (destinationBroker.isPresent()
+                                            && ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(config())) {
```

Comment:
I'm not sure why need to add param `destinationBroker`, if enable loadManagerExtension the validation already finish at line 646.